packages/palette/src/elements/LabeledRange/LabeledRange.tsx

```diff
@@ -57,22 +58,29 @@ export class LabeledRange extends React.Component<
   }
 
   render() {
-    const { formatter, label } = this.props
+    const { formatter, label, disabled, disabledText } = this.props
     const { min, max } = this.state
-
+    const useDisabledText = disabled && disabledText
```

Comment:
The "use" language is typically reserved for hooks

Reply:
ah ok, thanks

Reply:
pretty good word to reserve :-/